pkg/worker/worker.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/rs/zerolog"
 )
 
+var driversTable map[string]drivers.DriverFunc
```

Comment:
We probably don't want this to be a global table -- we probably want a `Worker` struct that has the property `driversTable`. We should have a good reason to use a global variable here, and I can see potential use-cases for wanting to instantiate multiple "workers" with different drivers. So rather than setting these in an `init` function we should set these in a constructor for a new `Worker`. What are your thoughts?

Reply:
Makes sense to me. By creating a `Worker` constructor, we might as well be able to encapsulate more information in the next iterations. Will make the changes and update.